src/blender_mcp/server.py

```
@@ -9,10 +9,52 @@
+
+    Returns None if the URL is safe, or an error message string otherwise.
+    """
+    parsed = urlparse(url)
+    hostname = parsed.hostname
+    if not hostname:
+        return "URL has no hostname"
+    try:
+        addr_infos = socket.getaddrinfo(hostname, parsed.port or 443)
+    except socket.gaierror:
+        return f"Could not resolve hostname: {hostname}"
+    for family, _, _, _, sockaddr in addr_infos:
```

Ruff (0.15.5)

[warning] Loop control variable `family` not used within loop body

Rename unused `family` to `_family`

(B007)